code/modules/events/special_antag_event.dm

```diff
@@ -34,7 +34,7 @@
 	fakeable = FALSE
 	var/role_name
 	var/antag_datum	//The datum of the antag E.G. /datum/antagonist/special/undercover
-	var/preference_type = ROLE_SPECIAL
+	var/preference_type = ROLE_TRAITOR
 	var/protected_jobs = list("Security Officer", "Warden", "Detective", "Head of Security", "Head of Personnel", "Chief Medical Officer", "Chief Engineer", "Research Director", "Captain", "Brig Physician")
 
 /datum/round_event/create_special_antag/start()
```

Comment:
I've heard reports from admins about this event. Can it occur right now?

Reply:
The fix just got merged